src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -850,7 +874,7 @@ public static class AkkaClusterHostingExtensions
             IMessageExtractor messageExtractor,
             ShardOptions shardOptions)
         {
-            shardOptions.Apply(builder);
+            shardOptions.DistributedData.Apply(builder);
```

Comment:
Just don't worry about DData settings. Please remove this entirely from the API. Let users configure DData separately if needed. Only stick to what's in the `ClusterShardingSetting`s class for this.

Reply:
how would the user set the `akka.cluster.sharding.distribute-data` settings then?

Reply:
If a situation arises where they would ever need to do it, they can do it through HOCON.